doc/nrf/ug_bt_coex.rst

```diff
@@ -0,0 +1,77 @@
+.. _ug_bt_coex:
+
+Using Bluetooth external radio coexistence
+##########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide describes how to add support for Bluetooth® coexistence to your application in |NCS|.
+
+.. _ug_bt_coex_overview:
+
+Overview
+********
+
+The coexistence feature can be used to reduce radio interference when multiple devices are located close to each other.
+This feature puts the Bluetooth stack under the control of a Packet Traffic Arbitrator (PTA) through a three-wire interface.
+The feature can only be used with the SoftDevice Controller, and only on nRF52 Series devices.
+
+The implementation is based on :ref:`nrfxlib:bluetooth_coex` which is integrated into nrfxlib's MPSL library.
+
+.. _ug_bt_coex_requirements:
+
+Enabling coexistence and MPSL
+*****************************
+
+Make sure that the following Kconfig options are enabled:
+
+   * :kconfig:`CONFIG_MPSL_CX`
+   * :kconfig:`CONFIG_MPSL_CX_BT`
+
+.. _ug_bt_coex_config:
+
+Configuring coexistence
+***********************
+
+Configuration is set using the devicetree (DTS).
+The MPSL coexistence sample :file:`samples/bluetooth/radio_coex_3wire` provides an example of adding the configuration as a devicetree overlay.
+For more information about devicetree overlays, see :ref:`zephyr:use-dt-overlays`.
+
+A sample configuration is provided below:
+
+   .. code-block::
+
+      / {
+        radio_coex: radio_coex_three_wire {
+          status = "okay";
+          compatible = "sdc-radio-coex-three-wire";
+          req-gpios =     <&gpio0 3 GPIO_ACTIVE_HIGH>; /* P0.03 */
+          pri-dir-gpios = <&gpio0 4 GPIO_ACTIVE_HIGH>; /* P0.04 */
+          grant-gpios =   <&gpio0 2 GPIO_ACTIVE_HIGH>; /* P0.02 */
+          type-delay-us = <8>;
+          radio-delay-us = <5>;
+          is-rx-active-level = <0>;
+          };
+      };
+
+Each element can be configured accordingly.
+Description of the elements:
```

Comment:
Why is this here? This information should be in the description fields within the binding itself. Then you can just use a link to the DT bindings index page here.

Reply:
Does this file (sdc-radio-coex-three-wire.yaml) need to be renamed (nordic,nrf-sdc-radio-coex-three-wire.yaml) to work with DT binding index?

Reply:
You can use the `:dtcompatible:` directive to point to any binding that has a compatible; grep around for examples.